.github/workflows/deployment_workflow.yml

```diff
@@ -0,0 +1,83 @@
+# Source: https://github.dev/IBM/actions-ibmcloud-iks
+
+name: Build and deploy to IBM Kubernetes
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  GITHUB_SHA: ${{ github.sha }}
+  IBM_CLOUD_API_KEY: ${{ secrets.IBM_CLOUD_API_KEY }}
+  IBM_CLOUD_REGION: eu-de
+  K8S_NAMESPACE: part2 # ADAPT!
+  IMAGE_NAME: se-booking-service-part2 # ADAPT!
+  IKS_CLUSTER: ${{ secrets.IKS_CLUSTER }} # name or id of cluster
+  DEPLOYMENT_NAME: se-booking-service-part2 # ADAPT!
+  INTERNAL_PORT: 8080 # ADAPT!
+  EXTERNAL_PORT: 32769 # ADAPT!
```

Comment:
The default node port range for Kubernetes is 30000 - 32767

Reply:
used 30069